event_sink_clickhouse/sinks/base_sink.py

```diff
@@ -62,3 +81,170 @@ def _send_clickhouse_request(self, request, expected_insert_rows=None):
             # ClickHouse can be configured not to return the metadata / summary we check above for
             # performance reasons. It's not critical, so we eat those here.
             return response
+
+
+class ModelBaseSink(BaseSink):
+    """
+    Base class for ClickHouse event sink, allows overwriting of default settings
+
+    This class is used for the model based event sink, which uses the Django ORM to write
+    events to ClickHouse.
+    """
+
+    unique_key = None
+    clickhouse_table_name = None
+    queryset = None
+    name = None
+    timestamp_field = None
+    serializer_class = None
+    model = None
```

Comment:
how do you intend to override these class attributes? should we explain that these variables should be defined when creating the child classes? It would be helpful as well to explain what each variable does.

Another thing, if those are constants for each child class, I think they should be in all uppercase

Reply:
If those variables are not defined, the class will raise an exception at instance creation. I will leave inline comments and add documentation on how to implement this on a plugin.

And for the naming, I was trying the same approach as Django generic classes does